web_ui/frontend/components/NamespaceTable.tsx

```diff
@@ -0,0 +1,82 @@
+import {Typography, Box, Button, ButtonProps} from '@mui/material';
+import React, {
+    useCallback,
+    useEffect,
+    useState
+} from "react";
+import {Skeleton} from "@mui/material";
+
+import DataTable, {Record} from "@/components/DataTable";
+import {TableCellOverflow, TableCellButton} from "@/components/Cell";
+
+
+interface Namespace extends Record {
+    id: number
+    prefix: string
+    pubKey: string
+    identity: string
+    adminMetadata: string
+}
+
+
+interface ServerTableProps {
+    type?: "cache" | "origin"
+}
+
+const NamespaceTable = ({type} : ServerTableProps) => {
+
+    const [data, setData] = useState<Namespace[] | undefined>(undefined);
+    const [error, setError] = useState<string | undefined>(undefined);
+
+    const keyToName = {
```

Comment:
Can we add `identity` to the table as well? I think we'd like to have it for admin to do approval work as the next step